Lib/configparser.py

```diff
@@ -757,6 +758,7 @@ def read(self, filenames, encoding=None):
             if isinstance(filename, os.PathLike):
                 filename = os.fspath(filename)
             read_ok.append(filename)
+        self._loaded_sources.extend(read_ok)
```

Comment:
```suggestion
            read_ok.append(filename)
            self._loaded_sources.append(filename)
```

Sorry this one is probably better because if we fail here for some reason, then we still want to say that we've read some sources (even if it failed) I think.